src/pages/user/order/Order.tsx

```diff
@@ -74,21 +123,73 @@ function Order() {
 
   const categoryRefs = useRef<Record<string, HTMLDivElement | null>>({});
 
+  const headerRef = useRef<HTMLDivElement>(null);
+  const [isShowNavBar, setIsShowNavBar] = useState(false);
+
   useEffect(() => {
     fetchWorkspace(workspaceId);
     fetchCategories();
+
+    const handleScroll = () => {
+      if (!headerRef.current) return;
+
+      const headerBottom = headerRef.current.getBoundingClientRect().bottom;
+
+      if (headerBottom - 65 <= 0) {
+        setIsShowNavBar(true);
+      } else {
+        setIsShowNavBar(false);
+      }
+    };
+
+    window.addEventListener('scroll', handleScroll);
+    return () => window.removeEventListener('scroll', handleScroll);
   }, []);
 
+  const onClickShare = async () => {
+    if (navigator.share) {
+      try {
+        await navigator.share({
+          title: document.title,
+          text: `${workspace.name}의 ${tableNo}번 테이블에서 주문하기기`,
+          url: window.location.href,
+        });
+      } catch (error) {
+        alert(`공유 실패: ${error}`);
+      }
+    } else {
+      alert('이 브라우저는 Web Share API를 지원하지 않습니다. 다른 브라우저를 사용해주세요.');
+    }
+  };
+
   return (
     <Container className={'order-container'}>
-      <Header className={'order-header'}>
-        <OrderImageSlider images={workspace.images} />
-        <AppLabel size={'medium'}>{workspace.name}</AppLabel>
+      <OrderStickyNavBar isShow={isShowNavBar}>
+        <LeftContainer>
+          <ArrowLeftButton />
+          <AppLabel color={Color.BLACK} size={20} style={{ fontWeight: '600' }}>
+            {workspace.name}
+          </AppLabel>
+        </LeftContainer>
+        <RightContainer>
+          <ShareButton onClick={onClickShare} />
+        </RightContainer>
+      </OrderStickyNavBar>
+
+      <OrderImageSlider images={workspace.images} />
+
+      <HeaderLabelContainer ref={headerRef}>
```

Comment:
```suggestion
      <Header ref={headerRef}>
```

아래서도 그냥 StickyHeader 정도로만 사용하고 있으니 위에도 header로만 써도 괜찮을 것 같아요~